server/api/emails.py

```diff
@@ -0,0 +1,40 @@
+from flask import Blueprint
+from config import SENDGRID_API_KEY
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+emails = Blueprint('emails', __name__, url_prefix='emails')
+
+
+@emails.route('/monthly-overview', methods=['GET'])
+def end_of_month_expenses():
+    message = Mail(
+        from_email='no-reply@receipttracker.com',
+        to_emails='user@test.com',
+        subject='Your monthly expense overview',
+        html_content='Monthly expense overview here')
+    try:
+        sg = SendGridAPIClient(SENDGRID_API_KEY)
+        response = sg.send(message)
+        print(response.status_code)
```

Comment:
if not needed anymore can we delete these print statements. or, make a reminder to delete it later :) want us to get in the habit of forgetting these logs

Reply:
okay will do! :)